.github/workflows/review-ready-label.yml

```diff
@@ -59,7 +58,7 @@ jobs:
           pr_search_result_length=$(echo "${pr_search_result}" | jq length)
           if [ "$pr_search_result_length" -ne 1 ]; then
               echo "Commit not found or in multiple PRs"
-              exit 1
+              exit 0
```

Comment:
hm, so we'll keep the exit message in place, and don't act at all if we hit that case. Is this really what we want?
Guess we can't test it until we merge, right?